src/main/resources/templates/java/test/gradle/projectTemplate/build.gradle

```
@@ -2,7 +2,7 @@ plugins {
     // %static-code-analysis-start%
     id 'checkstyle'
     id 'pmd'
-    id 'com.github.spotbugs' version '5.0.12'
+    id 'com.github.spotbugs' version '5.0.13'
```

```suggestion
    id 'com.github.spotbugs' version '5.0.12'
```
Changing dependency versions should be done in sync with https://github.com/ls1intum/artemis-maven-docker to make sure dependencies are not downloaded for each submission.

---

I was thinking about this, since we should not release a new version for this single dependency update. Since there are no issues with the current plugin version (at least not that I'm aware of) be can also just stay at the older version.